### PR TITLE
DFS/Bipartite (Silver) the great revegetation solution

### PR DESCRIPTION
**Pull request by TruaShamu**
1. Made new markdown file for the great revegetation (usaco920) with java solution
2. Added the new solution link to the problem constructor in bipartite/dfs

This is my first pull so if I did anything wrong please let me know

### CODE REVIEW

**Comment by frodakcin** [comment]
Thanks for contributing! Your solution file should match the format shown here: https://usaco.guide/general/contributing/#adding-a-solution

Specifically, you need the "header"

ex:
```
---
id: usaco-920
source: USACO Silver [year?] [Month?]
title: ???
author: ???
---
```

**Comment by maggie-j-liu** [comment]
Also, use tabs instead of spaces to indent.

**Comment by maggie-j-liu** [comment]
The change to tabs was reverted by the next commit...

**Comment by TruaShamu** [comment]
Sorry, it should be fixed now.
Thanks @dongliu0426 for helping me :)

**Comment by maggie-j-liu** [comment]
I still don't see it... did you commit?

**Comment by TruaShamu** [comment]
It's [here](https://usaco-guide-lhtmer38o-cpinitiative.vercel.app/solutions/usaco-920?lang=cpp)?

**Comment by dongliu0426** [comment]
> It's [here](https://usaco-guide-lhtmer38o-cpinitiative.vercel.app/solutions/usaco-920?lang=cpp)?

Maggie meant the commit changing spaces to tabs got reverted (i.e. [this commit](https://github.com/cpinitiative/usaco-guide/pull/941/commits/fe4f7e45d48429b3647379056c46f39c49a18a9e)).

**Comment by TruaShamu** [comment]
Sorry, I fixed the indentation now.

**Review by maggie-j-liu — CHANGES_REQUESTED** [review]

File: solutions/USACO feb19-the-great-revegetation.mdx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+		StringTokenizer st = new StringTokenizer(br.readLine());
+		int N = Integer.parseInt(st.nextToken());
+		int M = Integer.parseInt(st.nextToken());
+
+		type = new int[N + 1];
+		same = new ArrayList[N + 1];
+		diff = new ArrayList[N + 1];
+
+		//Initialize the adjacency lists.
+		for (int i = 0; i < N + 1; i++) {
```
Comment (dongliu0426) [inline solutions/USACO feb19-the-great-revegetation.mdx:38]:
```suggestion
		for (int i = 0; i <= N; i++) {
```
inconsistent formatting (used `for (int i = 0; i <= N; i++)` below) change this or the one below

**Review by dongliu0426 — CHANGES_REQUESTED** [review]

**Review by maggie-j-liu — APPROVED** [review]

**Pull request merged**